docs/sql/index.md

```diff
@@ -1,407 +0,0 @@
-
-| Example   | Description |
-| --------- | ----------- |
-| `1`       | An integer. |
-| `1.0`     | A float.    |
-| `'hello'` | A string.   |
-| `true`    | A boolean.  |
-
-#### Binary operators
-
-| Example | Description         |
-| ------- | ------------------- |
-| `1 > 2` | Integer comparison. |
-| `1 + 2` | Integer addition.   |
-
-#### Logical expressions
-
-Any expression which returns a boolean, i.e. `true` or `false`, is a logical expression.
-
-| Example          | Description                                                  |
-| ---------------- | ------------------------------------------------------------ |
-| `1 > 2`          | Integer comparison.                                          |
-| `1 + 2 == 3`     | Equality comparison between a constant and a computed value. |
-| `true AND false` | Boolean and.                                                 |
-| `true OR false`  | Boolean or.                                                  |
-| `NOT true`       | Boolean inverse.                                             |
-
-#### Function calls
-
-| Example         | Description                                        |
-| --------------- | -------------------------------------------------- |
-| `lower('JOHN')` | Apply the function `lower` to the string `'JOHN'`. |
-
-#### Table identifiers
-
-| Example       | Description               |
-| ------------- | ------------------------- |
-| `inventory`   | Refers to a table.        |
-| `"inventory"` | Refers to the same table. |
-
-#### Column references
-
-| Example                    | Description                                             |
-| -------------------------- | ------------------------------------------------------- |
-| `inventory_id`             | Refers to a column.                                     |
-| `"inventory_id"`           | Refers to the same column.                              |
-| `"inventory.inventory_id"` | Refers to the same column, explicitly naming its table. |
-
-#### Wildcards
-
-Special "star" expressions which select all the columns of a table.
-
-| Example       | Description                                             |
-| ------------- | ------------------------------------------------------- |
-| `*`           | Refers to all columns of a table identified by context. |
-| `inventory.*` | Refers to all columns of the `inventory` table.         |
-
-#### Parenthesized expressions
-
-Sub-expressions can be enclosed in parentheses for grouping and to override operator precedence.
-
-| Example       | Description             |
-| ------------- | ----------------------- |
-| `1 + (2 / 3)` | One plus a fraction.    |
-| `(1 + 2) / 3` | A sum divided by three. |
-
-### `CREATE TABLE`
-
-A `CREATE TABLE` statement creates a new, initially empty table in the database.
-
-The syntax of the `CREATE TABLE` statement is:
-
-> **CREATE TABLE** _table_name_ (_column_name_ _data_type_, ...);
-
-![create-table](/images/syntax/create_table.svg)
```

Comment:
Images point to the [spacetime-web](https://github.com/clockworklabs/spacetime-web) repo. Which means they don't render here.